### ISSUE TITLE
Blocking connections with a wrench

### ISSUE THREAD

**Issue by raoulvdberge** [raoulvdberge/refinedpipes#17]

**Comment by GalacticRuler** [comment]
Is there any sort of ETA on this at all?
Currently without this pipe networks take a ton of space and are massively complex.

**Comment by Krelnia** [comment]
Agreed. I'm having an issue right now where I have no other choice but to have the pipes next to each other but having them connected doesnt work with the setup

**Comment by vladmarica** [comment]
Personally, I would prefer it connections could be blocked using covers instead of having to hit pipes with a wrench.

**Comment by WhiteBoi1211** [comment]
Any ETA on this at all?
